vllm/engine/arg_utils.py

```diff
@@ -30,6 +30,7 @@ class EngineArgs:
     max_num_batched_tokens: Optional[int] = None
     max_num_seqs: int = 256
     max_paddings: int = 256
+    max_log_probs: int = 5
```

Comment:
Is `max_logprobs` better than this? 
And we can add comments for why default value is 5 (from OpenAI API Reference?).